docs/guides/self_hosting.md

```diff
@@ -0,0 +1,117 @@
+# [Experimental] Self Hosting
+**This guide is currently experimental. Instructions are subject to change as we improve self-hosting capabilities.**
+
+We provide a Helm chart that deploys LLM Engine to an Elastic Kubernetes Cluster (https://aws.amazon.com/eks/). This Helm chart should be configured to connect to dependencies (such as a postgresql database) that you may already have available in your environment.
+
+The only portions of the helm chart that are production ready are the parts that configure and manage LLM Server itself—not Postgres, IAM, etc.
```

Comment:
`LLM Engine itself`

Also I think a hyphen usually has a space on either side of it when used like this?